### PR TITLE
Add current frame filter

### LINKED ISSUE #499: Add option to restrict/filter annotation list to show only those in current frame only
Here, where we have the filtering options, it might be nice to have a toggle for "Current frame only" that filters annotations in the list to only show those in the current frame. The toggle would be here:

image

### PR DESCRIPTION
**Pull request by bruyeret**
Close #499

### CODE REVIEW

File: src/components/AnnotationBrowser/AnnotationFilters.vue [hunk 1]
```diff
@@ -58,6 +65,14 @@ export default class AnnotationFilters extends Vue {
     this.filterStore.setShapeFilter(filter);
   }
 
+  get currentFrameFilter() {
```
Comment (luciemac) [inline src/components/AnnotationBrowser/AnnotationFilters.vue:68]: I am no sure about this variable name.
What do you think of something like "onlyCurrentFrame"?
In my opinion, it does not feel right to have a variable named "xxxFilter" and typed as a boolean..

Comment (bruyeret) [reply src/components/AnnotationBrowser/AnnotationFilters.vue]: Yes indeed, it was bad naming!

**Comment by luciemac** [comment]
LGTM

**Review by arjunrajlab — APPROVED** [review]
Works great, LGTM!

**Pull request merged**